release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -3397,7 +3397,22 @@ link:https://access.redhat.com/solutions/6965038[{product-title} 4.8.45 containe
 ==== Bug fixes
 * Previously, `NetworkManager-wait-online.service` timed out preventing a connection to `coreos-installer` before it fetched the Ignition config files. With this update, `NetworkManager-wait-online.service` has more time to load allowing `coreos-installer` to fetch Ignition config files. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1983773#[*BZ#1983773*])
 
-[id="ocp-4-8-45-upgrading"]
-==== Upgrading
+[id="ocp-4-8-45-updating"]
+==== Updating
+
+To update an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-8-46"]
+=== RHBA-2022:5424 - {product-title} 4.8.46 bug fix update
+
+Issued: 2022-07-06
+
+{product-title} release 4.8.46 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:5424[RHBA-2022:5424] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:5423[RHBA-2022:5423] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6966480[{product-title} 4.8.46 container image list]
+
+==== Updating
 
-To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
```

Comment:
Is there a reason you removed "within a minor version by"?

Reply:
nvm i see lol

Reply:
Appreciate the update 😁. Sorry, I had to 🤣.